src/libmw/include/mw/models/tx/Output.h

```diff
@@ -0,0 +1,131 @@
+#pragma once
+
+// Copyright (c) 2018-2019 David Burkett
+// Distributed under the MIT software license, see the accompanying
+// file LICENSE or http://www.opensource.org/licenses/mit-license.php.
+
+#include <mw/models/tx/Features.h>
+#include <mw/models/tx/OutputId.h>
+#include <mw/models/crypto/BlindingFactor.h>
+#include <mw/models/crypto/Commitment.h>
+#include <mw/models/crypto/ProofData.h>
+#include <mw/models/crypto/RangeProof.h>
+#include <mw/models/crypto/SecretKey.h>
+#include <mw/models/crypto/SignedMessage.h>
+#include <mw/traits/Committed.h>
+#include <mw/traits/Serializable.h>
+
+// Forward Declarations
+class StealthAddress;
+
+////////////////////////////////////////
+// OUTPUT
+////////////////////////////////////////
+class Output :
+    public Traits::ICommitted,
+    public Traits::ISerializable
+{
+public:
+    //
+    // Constructors
+    //
+    Output(
+        Commitment&& commitment,
+        Features features,
+        PublicKey&& receiver_pubkey,
+        PublicKey&& key_exchange_pubkey,
+        uint8_t view_tag,
+        uint64_t masked_value,
+        BigInt<16>&& masked_nonce,
+        PublicKey&& sender_pubkey,
+        Signature&& signature,
+        const RangeProof::CPtr& pProof
+    );
+    Output(const Output& Output) = default;
+    Output(Output&& Output) noexcept = default;
+    Output() = default;
+
+    //
+    // Factory
+    //
+    static Output Create(
+        BlindingFactor& blind_out,
+        const Features& features,
+        const SecretKey& sender_privkey,
+        const StealthAddress& receiver_addr,
+        const uint64_t value
+    );
+
+    //
+    // Destructor
+    //
+    virtual ~Output() = default;
+
+    //
+    // Operators
+    //
+    Output& operator=(const Output& Output) = default;
```

Comment:
Why is the parameter capitalized on these 4 lines?

Reply:
Probably find/replace mistake. I'll fix it